docs/api/graphql_queries.md

```diff
@@ -553,6 +553,79 @@ Response:
 }
 ```
 
+To filter products based on content fields:
+
+```
+{
+  products {
+    all {
+      edges {
+        node {
+          fields {
+            ... on DressContentFields {
+              name
+              description {
+                plaintext
+              }
+            }
+            _all {
+              fieldDefIdentifier
+              value
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+To filter products based on attributes:
+
+
+```
+{
+  products {
+    single(code: "BLUELACE") {
+      attributes {
+        ... on DressAttributes {
+          measure {
+            reason
+          }
+          lace_color {
+            identifier
+            value
+          }
+        }
+      }
+    }
+  }
+}
+
+```
+ 
+If the attribute type cannot be found in the schema, the response is:
```

Comment:
Nitpicking: given we return two attributes, I would personally rephrase it to be more specific, something like:
```suggestion
If the attribute type cannot be found (`measure` in this case) in the schema, the response is:
```